backend/API/__init__.py

```diff
@@ -0,0 +1,39 @@
+import os
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_login import LoginManager
+
+# init SQLAlchemy so we can use it later in our models
+db = SQLAlchemy()
+upload_folder ="uploads\\"
```

Comment:
this kills linux versions (I now have a `uploads\` directory lol)

```suggestion
upload_folder ="uploads" +  os.pathsep
```